configs/envoy_double_proxy_v2.template.yaml

```diff
@@ -42,14 +43,18 @@
                 timeout: 20s
         http_filters:
         - name: envoy.health_check
-          config:
+          typed_config:
+            '@type': type.googleapis.com/envoy.config.filter.http.health_check.v2.HealthCheck
             pass_through_mode: false
-            endpoint: /healthcheck
-          name: envoy.buffer
-          config:
+            headers:
+              - exact_match: /healthcheck
```

Comment:
How come this changed?

Reply:
The healthcheck configuration has changed. It used to take a `endpoint: /healthcheck` but when you switch to the `Any` format healthcheck it requires the `headers` format - see https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/filter/http/health_check/v2/health_check.proto#envoy-api-msg-config-filter-http-health-check-v2-healthcheck